presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveStorageFormats.java

```diff
@@ -228,12 +231,26 @@ private static void setSessionProperties(Map<String, String> sessionProperties)
     {
         Connection connection = defaultQueryExecutor().getConnection();
         try {
-            PrestoConnection prestoConnection = connection.unwrap(PrestoConnection.class);
-            // create more than one split
-            prestoConnection.setSessionProperty("task_writer_count", "4");
-            prestoConnection.setSessionProperty("redistribute_writes", "false");
-            for (Map.Entry<String, String> sessionProperty : sessionProperties.entrySet()) {
-                prestoConnection.setSessionProperty(sessionProperty.getKey(), sessionProperty.getValue());
+            if (usingFacebookJdbcDriver()) {
```

Comment:
Good idea to extract setSessionProperty() so that it works with Simba or FB driver.  I will do that.

Will also take another look at using*JdbcDriver() methods.